src/ActionModal/components/ModalWithdrawRewards.vue

```diff
@@ -13,7 +13,10 @@
     :rewards="rewards"
     :disable="validatorsWithRewards"
   >
-    <span class="form-message notice withdraw-limit">
+    <span
+      v-if="currentNetwork.network_type !== 'polkadot'"
```

Comment:
```suggestion
      v-if="currentNetwork.network_type === 'cosmos'"
```

This is an example for a positive check vs a negative check :D. If I would add type "ethereum" later, this will still be correct.